src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,282 @@
+{{site.data.alerts.end}}
+
+</section>
+
+## Set up a cluster
 
-1.  Use the {{ site.data.products.db }} Console to [create your {{ site.data.products.dedicated }} cluster](create-your-cluster.html) on AWS in the same region as your application.
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+1. Use the {{ site.data.products.db }} Console to [create your {{ site.data.products.dedicated }} cluster](create-your-cluster.html) on AWS in the same region as your application.
 
     {{site.data.alerts.callout_info}}
     If you have a multi-region cluster, you will have to create a PrivateLink connection for each region you are operating in.
     {{site.data.alerts.end}}
 
-1.  Navigate to the **Networking** page.
-1.  Select the **PrivateLink** tab.
-1.  Click **Set up a PrivateLink connection** to open the connection modal.
+1. Navigate to the **Networking** page.
+1. Select the **PrivateLink** tab.
+1. Click **Add Connection** to open the connection modal.
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+1. Use the {{ site.data.products.db }} Console to [create a multi-region {{ site.data.products.serverless }} cluster](create-a-serverless-cluster.html) on AWS in the same region as your application.
+
+    {{site.data.alerts.callout_info}}
+    **Multi-region for {{ site.data.products.serverless }} is in [preview](../{{site.versions["stable"]}}/cockroachdb-feature-availability.html)** and subject to change. You cannot currently add or remove regions once a cluster has been created. To share feedback and/or issues, contact [Support](https://support.cockroachlabs.com/).
+    {{site.data.alerts.end}}
+
+1. Navigate to the **Networking** page.
+1. Select the **PrivateLink** tab. PrivateLink connections that have already been configured are shown. A maximum of one PrivateLink connection can be configured.
+1. To add a new private connection, click **Add Connection** to open the connection modal.
+
+</section>
 
-### Create an AWS endpoint
+## Create an AWS endpoint
+
+{% capture security_group_substeps %}
+    <ul><li>In the **Security group name** field, enter a name for the security group.</li>
+    <li>In the **Description** field, enter a description for the security group.</li>
+    <li>From the **VPC** dropdown, select the VPC you chose in Step 4.</li>
+    <li><ul><li>In the **Inbound rules** section, click **Add rule**.</li><li>Enter *26257* in the **Port range** field.</li><li>In the **Source** field, enter the CIDR range from Step 5.</li></ul></li>
+    <li>Click **Create security group**.</li></ul>
+{% endcapture %}
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+1. Select the region to create a connection in.
```

Comment:
This list breaks when it's in a filter-content section but not when I comment out the sections, it seems like it is ignore the `markdown="1"`. It breaks even before I capture the sublist into a variable. Hoping others can spot what I'm doing wrong.